<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>576</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>ΕΚΤΥΠΩΣΗ ΤΑΜΠΕΛΑΚΙΑ ΑΙΤΗΣΕΩΝ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_browser">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>browser</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Α_ΤΜΗΜΑ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_print">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>450</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>print</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_browser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_posothta">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>450</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>450</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ΠΟΣΟΤΗΤΑ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_paratirisis">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>330</y>
      <width>551</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="data_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>169</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ΠΑΡΑΤΗΡΗΣΕΙΣ:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_green">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GREEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_about">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>about   set printer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="data_11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ΟΝΟΜΑ ΑΡΧΕΙΟΥ:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>70</y>
      <width>201</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="data_8">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Σ. ΚΩΔΙΚΩΝ:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="data_aa"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="data_9">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Σ. ΤΕΜΑΧΙΩΝ:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="data_count_aa"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>71</y>
      <width>481</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="data_1">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ΑΙΤΗΣΗ ΚΟΠΗΣ:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="data_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ΑΡ. ΕΝΤΟΛΗΣ:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="data_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ΤΟΜΕΑΣ:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="line_3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="data_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ΗΜΕΡ. ΑΙΤΗΣΗΣ:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="line_4"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="data_5">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ΗΜΕΡ. ΠΑΡΑΔΟΣΗΣ:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="line_5"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="data_6">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ΥΠΕΥΘΥΝΟΣ:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="line_6"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="data_7">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ΠΑΡΑΔΟΣΗ:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="line_7"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="line_1"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="line_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
